daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -280,6 +280,43 @@ async function start() {
 }
 ```
 
+#### Bulk Subscribe to messages
+
+Dapr [supports bulk subscribing](https://v1-10.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/) a topic.
+
```

Comment:
In general it feels redundant to have both these methods? Can't we make these parameters optional?

Also, can we name them `subscribeBulk` instead? making the API become: `server.pubsub.subscribeBulk` which follows the SDK a bit more

Reply:
Done